src/components/ebay-expand-button/index.marko

```diff
@@ -11,7 +11,8 @@ static var ignoredAttributes = [
     "truncate",
     "fixedHeight",
     "partiallyDisabled",
-    "toJSON"
+    "toJSON",
```

Comment:
Can you swap this, have `toJSON` be the last one since that's an internal variable.